src/main/resources/assets/css/main.less

```diff
@@ -406,9 +410,9 @@ fieldset[disabled] .btn-info.active {
   line-height: 1.5;
 }
 
-.pagination > li > a:hover,
+.pagination > li > button:hover,
 .pagination > li > span:hover,
-.pagination > li > a:focus,
+.pagination > li > button:focus,
```

Comment:
We need to add the padding that comes with the a tag normally here: `padding: 5px 10px;`